geode-core/src/test/java/org/apache/geode/distributed/internal/DistributionMessageTest.java

```diff
@@ -35,4 +39,12 @@ public void shouldBeMockable() throws Exception {
 
     verify(mockDistributionMessage, times(1)).setReplySender(mockReplySender);
   }
+
```

Comment:
is/was there a failing test that caught the underlying issue?

Reply:
No, this was in thread dumps when I was debugging an application with disable-tcp=true.